README.md

```diff
@@ -1,410 +1,158 @@
 # Appeal Planning Decision
 
-Monorepo for all PINS Appeal planning decision services.
+Appeal A Planning Decision Service Front Office monorepo
 
-> Note: some documentation also lives in [docs](./docs).
+- Please follow the [commit strategy](./docs/commit-strategy.md)
+- Please follow the repo's [best practice](./docs/best-practice.md)
 
-## TL;DR
+This repo is designed to provide a front end for various users involved in making and managing an appeal made to the [Planning Inspectorate](https://www.gov.uk/government/organisations/planning-inspectorate)
 
-- create an `.env` file in `./packages/database` (you can copy the `.env.example`) 
-- `npm ci` from root directory
-- `make serve`
-- once the server is running, run `npm run populate-db` to populate the Local Planning Authorities database
-- Appellant journey: go to [localhost:9003/appeal/new-saved-appeal](http://localhost:9003/appeal/new-saved-appeal) - if returning to a saved appeal, you can use appellant1@planninginspectorate.gov.uk (code: `12345`) to access seed data.
-- LPA dashboard: Go to [localhost:9003/manage-appeals/your-email-address](http://localhost:9003/manage-appeals/your-email-address). Sign in with email address admin1@planninginspectorate.gov.uk and code `12345`.
+**Table Of Contents**
 
-Note: when starting a new appeal as an appellant, if you cannot progress beyond the 'before you start' page without encountering an error page (and you see _Database &#96;pins_front_office_development&#96; does not exist on the database server at &#96;mssql:1433&#96;._ in the API logs), run `npm run db:migrate:dev` per [Database Server Setup](#database-server-setup) and refresh the page.
+1. [Appeals process](#Appeals-process)
+1. [Running locally](#Running-locally)
+1. [Additional Tools](#Additional-Tools)
+1. [Architecture](#Architecture)
+1. [Further information](#Further-information)
+1. [Related Repos](#Related-Repos)
 
-## Pre-requisites
+## Appeals process
 
-- [NodeJS v20](https://nodejs.org/en/download/)
-- [Docker](https://docs.docker.com/get-docker/)
-- [Docker Compose](https://docs.docker.com/compose/install/)
+- A member of public contacts the Planning Inspectorate to make an appeal a decision made by their Local Authority, they become an 'Appellant'
```

Comment:
unclear grammar - should be "to make an appeal against a decision" or "to appeal against a decision"

Reply:
also, maybe put "their Local Authority (also referred to as an LPA)" as this is what is found in codebase